backend/curation/api/v1/curation/collections/common.py

```diff
@@ -202,13 +202,18 @@ def reshape_dataset_for_curation_api(
     dataset_version: DatasetVersion,
     use_canonical_url: bool,
     preview=False,
+    index=False,
```

Comment:
going to tackle a refactor in a separate to see if we can avoid adding this new flag, otherwise this seems like the path of least resistance for the functional change

Reply:
Agreed